models/User.ts

```diff
@@ -112,18 +115,32 @@ export default class User implements KongUser, GovDeliveryUser {
 
     this.apiList = this.apis ? this.apis.split(',') : [];
 
-    const isApplyingForInternal = INTERNAL_ONLY_APIS.some(api => this.apiList.includes(api));
+    this.isApplyingForInternal = INTERNAL_ONLY_APIS.some(api => this.apiList.includes(api));
     const hasVAEmail = this.email.endsWith('va.gov') || this.vaEmail?.endsWith('va.gov');
-
-    if (isApplyingForInternal && !hasVAEmail) {
+    if (this.isApplyingForInternal && !hasVAEmail) {
       throw new Error('Applying for internal api without VA email');
```

Comment:
does this bubble up and become visible to the consumer?  if so, then "api" should probably be "API"